spec/app/models/mdm/service_spec.rb

```diff
@@ -71,6 +72,27 @@
         expect(svc).to receive(:normalize_host_os)
         svc.run_callbacks(:save)
       end
+      it 'should include recog data when there is a match' do
```

Comment:
Please put a blank line between each `it do end`.  `it { }` don't require blank lines between them.